pages/glossary/default-page-language.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Default Page Language
+key: default-page-language
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+---
+
+The default page language is the _most common language_ used in a [web page][]. If there are no _words_ in the page, the default language is undefined.
+
+The _most common language_ is determined by counting the number of _words_ in the [web page][] that are part of any of the languages in the [language subtag registry][]. The same word can be part of multiple languages. If the number of words is the same, the default language is undefined.
+
+The _words_ used in the following texts should be counted for the _most common language_:
+
+- **page title**: The value of the [document title][] of the [document][] in the [top-level browsing context][].
+- **text nodes**: The value of any [text nodes][] that is [visible][] or [included in the accessibility tree][]
+- **accessible text**: The [accessible name][] and [accessible description][] of any element included in the [accessibility tree][]
+
+**Exception**: Words in text that comes from nodes that have an [ancestor][] in the [flat tree][] with a non-empty (`""`) `lang` attribute, except for the [root node][].
```

Comment:
```suggestion
**Exception**: Do not count words in text that comes from nodes that have an [ancestor][] in the [flat tree][] which has a non-empty (`""`) `lang` attribute and is not the [root node][].
```
The original sentence is OK-ish but I feel it has two problems: 1/ not a sentence (no verb, …) so adding the "do not count" part; 2/ "except for the root node" was separated from what it qualifies ("ancestor in the flat tree"), making it look like it apply to the full nominal group ("nodes that have an ancestor with a non-empty lang attribute").

Not super happy with what I'm suggesting either, feels very "heavy" sentence for something relatively simple…